reactive/k8s.py

```diff
@@ -76,7 +76,8 @@ def render_files(reldata):
     if not os.path.exists(rendered_manifest_dir):
         os.makedirs(rendered_manifest_dir)
     # Add the manifest directory so the docker-compose file can have.
-    context.update({'manifest_directory': rendered_manifest_dir})
+    context.update({'manifest_directory': rendered_manifest_dir,
+                    'private_address': hookenv.unit_get('private-address')})
```

Comment:
In the last code I named this value as api_private_address to disambiguate the values and addresses going around.